projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/node-types/object-node-flat/object-node-flat.component.html

```diff
@@ -87,9 +87,12 @@
   <ng-container *ngIf="showKnownProperties">
     <ng-container *ngFor="let prop of schema.properties | keyvalue">
       <div class="node-container add-button add-prop-button" *ngIf="model[prop.key] === undefined">
-        <span class="json-editor--vertical-separators" *ngFor="let separator of indentationArray"></span>
-
-        <div class="node" [style.marginLeft]="level === 0 ? '20px' : '0'" (click)="addSchemaProperty(prop.key)">
+        <span class="json-editor--vertical-separator" *ngFor="let separator of indentationArray"></span>
```

Comment:
the previous class was "json-editor--vertical-separators" and the new one is "json-editor--vertical-separator", so the last "s" was removed, is it correct?

Reply:
yes it had a typo , the correct class name is json-editor--vertical-separator